apps/comps/components/timeline-chart/chart-template.tsx

```diff
@@ -377,43 +388,147 @@ export const MetricTimelineChart: React.FC<MetricTimelineChartProps> = ({
-            <div className="text-center">
-              <h3 className="mb-3 text-2xl font-bold uppercase tracking-wider text-white">
-                Competition Starts In
-              </h3>
-              <CountdownClock
-                targetDate={new Date(startDate)}
-                className="text-5xl font-bold tabular-nums text-white"
-                showDuration={true}
-                onFinish={() => {
-                  // Refresh to update competition status
-                  window.location.reload();
-                }}
-              />
+        <div className="h-150 flex w-full flex-col items-center justify-center">
+          <div className="flex w-full max-w-2xl flex-col items-center justify-center space-y-8 px-8 py-16 text-center">
+            <div className="space-y-4">
+              <h2 className="text-primary-foreground text-3xl font-semibold tracking-wide">
+                Competition Details Pending
+              </h2>
             </div>
-            <p className="text-md text-secondary-foreground">
-              {formatDate(startDate)}
+            <p className="text-secondary-foreground text-lg">
+              Check back soon for the competition&apos;s start time,
+              requirements, and boosting details.
             </p>
           </div>
         </div>
       );
     }
 
-    // Default message for no data
+    // Start date exists - show countdown(s) for competition and boosting
+    const showCompetitionCountdown = isFuture(startDate);
+    const showBoostingCountdown = boostStartDate && isFuture(boostStartDate);
+    const showBothCountdowns =
+      showCompetitionCountdown && showBoostingCountdown;
+
+    return (
+      <div className="h-150 flex w-full flex-col items-center justify-center">
+        <div className="flex w-full max-w-7xl flex-col items-center justify-center px-4 py-16">
+          {/* Show countdown(s) side by side if both exist, otherwise centered */}
+          {(showCompetitionCountdown || showBoostingCountdown) && (
+            <div
+              className={`grid w-full ${
+                showBothCountdowns
+                  ? "relative grid-cols-1 gap-16 sm:grid-cols-2"
+                  : "grid-cols-1"
+              }`}
+            >
+              {/* Boost start countdown */}
+              {showBoostingCountdown && (
+                <div className="flex flex-col items-center space-y-8 text-center">
+                  <div className="space-y-2">
+                    <h2 className="text-primary-foreground text-3xl font-semibold uppercase tracking-wide">
+                      Boosting Opens In
+                    </h2>
+                  </div>
+                  <CountdownClock
+                    targetDate={boostStartDate}
+                    className="text-primary text-6xl font-bold"
+                    showDuration={true}
+                    onFinish={() => {
+                      if (!hasRefreshed.current) {
```

Comment:
the only logic change was using this instead of `window.location.reload();`.

the other changes just add different things we can display based on the comp's state